scylla/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "display_name": "Scylla",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "scylla",
+  "metric_prefix": "scylla.",
+  "metric_to_check": "scylla.scylladb.current_version",
```

Comment:
This will never be collected since it's defined as the `metadata_metric_name`

Reply:
Does that mean `metric_to_check` can just be omitted or there should be a different metric referenced?

Reply:
Another default metric should be referenced. `metric_to_check` automatically has the integration marked as `Installed` in the tile when it's collected, so it should be one that is collected by default.